### DISCUSSION TITLE
arm64 dockerfile

### DISCUSSION THREAD

**Discussion by changboon** [almond-sh/almond#933] [Q&A]
Hi team,

is it possible to advise on how I can modify the dockerfile to build an arm64 version of the image?

Thanks

**Comment by alexarchambault** [comment]
I'm not sure. You mean [this Dockerfile](https://github.com/almond-sh/almond/blob/260ce5a37886b536de85254bcd5df1f1336b14b6/Dockerfile)? I guess you'll need a different set of base images. When do things fail for you?
